### PR TITLE
Add more doc comments for CRDs

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This PR simply adds some documentation for fields where I knew what to write. Nothing except for comments was changed.

/kind documentation
**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/kubermatic/v1/cluster.go [hunk 1]
```diff
@@ -104,11 +104,17 @@ var ProtectedClusterLabels = sets.NewString(WorkerNameLabelKey, ProjectIDLabelKe
 // +kubebuilder:printcolumn:JSONPath=".metadata.creationTimestamp",name="Age",type="date"
 
 // Cluster represents a Kubermatic Kubernetes Platform user cluster.
+// Cluster objects exist on Seed clusters and each user cluster consists
+// of a namespace containing the Kubernetes control plane, additional
+// pods (like Prometheus).
```
Comment (lsviben) [inline pkg/apis/kubermatic/v1/cluster.go]:
```suggestion
// of a namespace containing the Kubernetes control plane and additional
// pods (like Prometheus).
```

File: pkg/apis/kubermatic/v1/preset.go [hunk 2]
```diff
@@ -37,7 +37,10 @@ type PresetList struct {
 // +kubebuilder:object:generate=true
 // +kubebuilder:object:root=true
 
-// Preset is the type representing a Preset.
+// Preset is the type representing a Preset. Presets are preconfigured
+// cloud provider credentials that can be applied to new clusters. This
+// frees end users from having to know the actual credentials used for
+// their clusters.
```
Comment (lsviben) [inline pkg/apis/kubermatic/v1/preset.go]: Maybe something like this to get rid of `Preset=Preset` sentance?

```suggestion
// Preset represents preconfigured
// cloud provider credentials that can be applied to new clusters. This
// frees end users from having to know the actual credentials used for
// their clusters.
```

**Review by lsviben — COMMENTED** [review]
some minor suggestions

**Review by embik — APPROVED** [review]

**Pull request merged**